### PR TITLE
Plugin/transponder: add tslc in AdsbVehicle struct

### PR DESCRIPTION
**Pull request by chep**
Time since last communication is missing in AdsbVehicle struct. I need it to remove aircrafts in my GUI so I made this patch.

### CODE REVIEW

File: src/mavsdk/plugins/transponder/include/plugins/transponder/transponder.h [hunk 1]
```diff
@@ -107,6 +107,7 @@ class Transponder : public PluginBase {
         std::string callsign{}; /**< @brief The callsign */
         AdsbEmitterType emitter_type{}; /**< @brief ADSB emitter type. */
         uint32_t squawk{}; /**< @brief Squawk code. */
+        uint8_t tslc{}; /**< @brief Time Since Last Communication. */
```
Comment (JonasVautherin) [inline src/mavsdk/plugins/transponder/include/plugins/transponder/transponder.h:110]: This file is auto-generated, you have to make that change in the corresponding proto file :+1:

**Review by JonasVautherin — CHANGES_REQUESTED** [review]

File: src/mavsdk/plugins/transponder/include/plugins/transponder/transponder.h (hunk 1 above)
Comment (chep) [reply src/mavsdk/plugins/transponder/include/plugins/transponder/transponder.h:110]: I created https://github.com/mavlink/MAVSDK-Proto/pull/280

How does it work? You merge pull request in MAVSDK-proto, re-generate files and commit them in MAVSDK?

Comment (JonasVautherin) [reply src/mavsdk/plugins/transponder/include/plugins/transponder/transponder.h:110]: Yes :+1:. If that was about bigger changes, you would re-generate files here without merging into MAVSDK-proto (you can point the proto submodule to your branch), but in that case it's simple enough we can merge it directly in MAVSDK-proto.

**Comment by julianoes** [comment]
Oops @chep, I wanted to finish this, so I amended your commit and had it all ready. Then, by stupidity, pushed main instead of your main branch onto your main branch using:
```
git push -u chep main -f
```
When I should have done:
```
git push -u chep chep-main:main -f
``` 
`chep-main` is my local branch of `chep/main`

So then GitHub decided to close this PR and now I don't have permission to re-push :see_no_evil:.

So, I'll create a new PR and use your commit there, sorry about that.

**Pull request closed without merging**